doc/glossary.rst

```diff
@@ -1218,6 +1235,11 @@ Methods
         (i.e. training and test data together) before further modelling, as
         this results in :term:`data leakage`.
 
+    ``fit_resample``
+        A method on :term:`resamplers` which fits the estimator on a passed
+        dataset, and returns a new dataset. In the new dataset, samples may be
```

Comment:
Needs more spec on return value.

Reply:
Should we wait on the SLEP for this? For now I could implement that any kwargs that are numpy arrays (and share shape[0] with X and y) will be resampled aswell.

Reply:
Well this implementation supports the slep. Please index all the kwargs in outlier rejection, using safe_indexing. No need to check if they are numpy arrays.